src/cli/cli_srp_client.cpp

```diff
@@ -172,6 +280,33 @@ template <> otError SrpClient::Process<Cmd("host")>(Arg aArgs[])
                 }
             }
         }
+        /**
+         * @cli srp client host address (set)
+         * @code
+         * srp client host address auto
+         * Done
+         * @endcode
+         * @code
+         * srp client host address fd00::cafe
+         * Done
+         * @endcode
+         * @cparam client host address [auto|@ca{address...}]
+         *   * Use the `auto` parameter to enable auto host address mode.
+         *     When enabled, the client automatically uses all Thread `netif`
+         *     unicast addresses except for link-local and mesh-local
+         *     addresses. If there is no valid address, the mesh local
+         *     EID address gets added. The SRP client automatically
+         *     re-registers if addresses on the Thread `netif` are
+         *     added or removed.
+         *   * Explicitly specify the list of host addresses, separating
+         *     each address by a comma. You can set this list while the client is
+         *     running. This will also disable auto host address mode.
```

Comment:
We can use space in between addresses (when there are more than one), we cannot use comma.